pages/instructorAssessments/instructorAssessments.sql

```diff
@@ -46,7 +46,7 @@ SELECT
         LAG (CASE WHEN $assessments_group_by = 'Set' THEN aset.id ELSE am.id END)
         OVER (
             PARTITION BY (CASE WHEN $assessments_group_by = 'Set' THEN aset.id ELSE am.id END)
-            ORDER BY aset.id, a.order_by, a.id NULLS FIRST
+            ORDER BY aset.number, a.order_by, a.id NULLS FIRST
```

Comment:
Incidentally, I believe the NULLS FIRST is unnecessary here, as none of these values can be NULL. @nwalters512 correctly raised this point during review in https://github.com/PrairieLearn/PrairieLearn/pull/4360/files#r783535458 but that was never resolved.

Reply:
Great, I'll remove it so as not to confuse ourselves in the future 🙂

Reply:
Removed in 3e2955c.